lib/presentation/ui/contributors/contributors_page.dart

```diff
@@ -0,0 +1,113 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class Contributors extends StatefulWidget {
+  @override
+  _ContributorsState createState() => _ContributorsState();
+}
+
+class _ContributorsState extends State<Contributors> {
+  List<Contributor> _contributors;
+
+  @override
+  void initState() {
+    super.initState();
+    _loadContributors();
+  }
+
+  _loadContributors() async {
+    var jsonListContributors = jsonDecode(await DefaultAssetBundle.of(context)
+        .loadString('assets/data/contributors.json'));
+
+    List<Contributor> contributors = List<Contributor>.from(
+        jsonListContributors.map((m) => Contributor.fromJson(m)).toList());
+
+    this.setState(() {
+      _contributors = contributors;
+    });
+  }
+
+  void _openContributorPage(String url) async {
+    if (await canLaunch(url)) {
+      await launch(url);
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Contribuidores"),
+      ),
+      body: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Text(
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam tincidunt."),
+          ),
+          Expanded(
+            child: ListView.separated(
```

Comment:
You are receiving here data asynchronously, can you use a StreamBuilder or FutureBuilder? (when refactoring to BLoC)